File: tuts/003-s3-gettingstarted/php/GettingStartedScenario.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

require 'vendor/autoload.php';

use Aws\S3\S3Client;

$client = new S3Client(['region' => 'us-east-1']);
echo "Running S3 getting started scenario...\n";
// TODO: setup, interact, teardown
echo "Scenario complete.\n";

File: tuts/003-s3-gettingstarted/cpp/getting_started_scenario.cpp
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

#include <aws/core/Aws.h>
#include <aws/s3/S3Client.h>
#include "s3_wrapper.h"
#include <iostream>

int main(int argc, char *argv[]) {
    Aws::SDKOptions options;
    Aws::InitAPI(options);
    {
        Aws::S3::S3Client client;
        std::cout << "Running S3 getting started scenario..." << std::endl;
        // TODO: setup, interact, teardown
    }
    Aws::ShutdownAPI(options);
    return 0;
}
